scm-ui/ui-webapp/src/containers/Main.tsx

```diff
@@ -77,7 +77,8 @@ class Main extends React.Component<Props> {
             <Route path="/logout" component={Logout} />
             <Redirect exact strict from="/repos" to="/repos/" />
             <ProtectedRoute exact path="/repos/" component={Overview} authenticated={authenticated} />
-            <ProtectedRoute exact path="/repos/create" component={Create} authenticated={authenticated} />
+            <ProtectedRoute exact path="/repos/create" component={AddRepository} authenticated={authenticated} />
+            <ProtectedRoute exact path="/repos/import" component={AddRepository} authenticated={authenticated} />
```

Comment:
I'm not sure, but i think two different components for the routes would be easier.